system-design/app-interoperability/application-package-definition.md

```diff
@@ -75,6 +77,34 @@ deploymentProfiles:
           repository: oci://northstarida.azurecr.io/charts/hello-world
           revision: 1.0.1
           wait: true
+parameters:
+  greeting:
+    value: Hello
+    targets:
+    - pointer: global.config.appGreeting
+      components: ["hello-world"]
+  greetingTarget:
```

Comment:
I suggest to choose different parameter name here as "greetingTarget" confuses the reader with the "targets" key. Maybe use "greetingAddress"?